app/models/patient.rb

```diff
@@ -1305,7 +1310,7 @@ def monitoring_history_edit(history_data, diff_state)
       when :symptom_onset
         History.symptom_onset(history_data)
       when :case_status
-        History.case_status(history_data, diff_state)
+        History.case_status(has_updated_monitoring ? history_data.except(:reason) : history_data, diff_state)
```

Comment:
I'm reading this as covering the case where `monitoring` and `case_status` are both edited as part of the same action. Where in the UI can I do this? From the patient show page it's seeming like I can only edit one at a time so I can't trigger this logic from that page @DPC15038